Makefile

```diff
@@ -57,6 +62,15 @@ run-local: install-apis ## Install CRDs and run provisioner locally
 	kubectl create namespace rukpak-system
 	$(Q)go run provisioner/k8s/main.go
 
+deploy: install-apis ## Deploy the operator to the current cluster
+	kubectl apply -f provisioner/k8s/manifests
+
+run: build-local-container load-image deploy
```

Comment:
I think this is reasonable while we iterate but we may need to segment this to be provisioner-specific, or rethink how we install rukpak provisioners long term.